common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -94,4 +94,14 @@ trait ABTestSwitches {
     sellByDate = Some(LocalDate.of(2023, 1, 30)),
     exposeClientSide = true,
   )
+
+  Switch(
+    ABTests,
+    "ab-no-carrot-ads-near-newsletter-signup-blocks",
+    "Test the impact of preventing spacefinder from positioning carrot ads near newsletter signup blocks",
+    owners = Seq(Owner.withEmail("commercial.dev@theguardian.com")),
+    safeState = Off,
+    sellByDate = Some(LocalDate.of(2023, 1, 31)),
```

Comment:
The switch expiry and the test expiry and slightly different. This probably isn't a major issue given in all likelihood the test will finish well before then. However if we need to use it on exactly those days it could be a head scratcher!

Reply:
Great shout, I'll amend this